pkgs/development/tools/misc/c2ffi/default.nix

```diff
@@ -12,7 +12,7 @@ in
 
 llvmPackages.stdenv.mkDerivation {
   pname = "c2ffi-${c2ffiBranch}";
-  version = "unstable-2021-06-15";
+  version = "11.0.0.0-unstable-2021-06-15";
```

Comment:
This ends up with 2 LLVM 11 version numbers in its full name, due to the previous tag already being a base LLVM version. The one in `pname` can likely be dropped.

`c2ffi-llvm-11.0.0-11.0.0.0-unstable-2021-06-15`